packages/rs-drive-abci/src/platform_types/mod.rs

```diff
@@ -0,0 +1,60 @@
+// MIT LICENSE
```

Comment:
The concept of platform type is confusing and not clear. Part of these types are used in the execution part of them in ABCI. Execution types should be moved into execution modules